tools/bliss-to-ogg-zip.py

```diff
@@ -209,9 +336,12 @@ def main():
       if os.path.exists(dest_filename):
         print("already exists, delete: %s" % os.path.basename(dest_filename))
         os.remove(dest_filename)
-      cmd = ["ffmpeg", "-i", rec_filename, "-ss", str(seq.start_time), "-t", str(duration), dest_filename]
+      cmd = ["ffmpeg", "-i", source_filename, "-ss", str(start_time), "-t", str(duration), dest_filename,
+             "-loglevel", args.ffmpeg_loglevel]
       print("$ %s" % " ".join(cmd))
       check_call(cmd)
+    if args.sprint_cache:
+      os.remove(wav_tmp_filename)
```

Comment:
As an additional check, maybe you load the Ogg file now, and verify again that the number of frames is correct. Because I'm not 100% whether the wav->ogg ffmpeg encoding keeps exactly the same number of frames. (@JackTemaki maybe you know this?)

Reply:
I added a check and in a short test it looks like the number of frames is always correct.